### DISCUSSION TITLE
Window and taskbar problem after using Windows magnifying glass

### DISCUSSION THREAD

**Discussion by jaminchew** [valinet/ExplorerPatcher#2362] [Q&A]
Every time after I use the built in magnifying glass app, any full screen windows will no longer be adhere to the edge of the taskbar. Essentially ignoring the taskbar. The bottom edges of full screen apps will either be behind, or hide taskbar.  I have to restart Windows Explorer.exe to fix the issue. This has been a problem through at least the last few versions of the Explorer Patcher. 

Examples:
Windows Explorer behind the taskbar.
Notepad.exe is in front of the taskbar.
Internet browsers are in front of the taskbar
No full screen apps will adhere to the taskbar edge.

Windows 11 Pro Version 22H2 22621.2428
Explorer Patcher Version 22621.2428.59.1

Anyone else have this problem? Any fixes?

**Comment by pyrates999** [comment]
Please include what settings you changed in explorer patcher properties application.
